ingest/ingest_pipeline.py

```diff
@@ -33,42 +33,43 @@
 # Ingest mtx files
 python ingest_pipeline.py --study-id 5d276a50421aa9117c982845 --study-file-id 5dd5ae25421aa910a723a337 ingest_expression --taxon-name 'Homo sapiens' --taxon-common-name humans --matrix-file ../tests/data/matrix.mtx --matrix-file-type mtx --gene-file ../tests/data/genes.tsv --barcode-file ../tests/data/barcodes.tsv
 """
+import datetime
 import json
 import logging
 import os
+import pprint
 import re
 import sys
 from contextlib import nullcontext
 from typing import Dict, Generator, List, Tuple, Union  # noqa: F401
 
 # import google.cloud.logging
 from bson.objectid import ObjectId
-
-# For tracing
-from opencensus.ext.stackdriver.trace_exporter import StackdriverExporter
-from opencensus.trace.samplers import AlwaysOnSampler
-from opencensus.trace.tracer import Tracer
-from pymongo import InsertOne, MongoClient
-from pymongo.errors import BulkWriteError
-
+from cell_metadata import CellMetadata
+from cli_parser import create_parser, validate_arguments
+from clusters import Clusters
+from expression_files.dense import Dense
+from expression_files.dense_command import DenseCommand
 # from google.cloud.logging.resource import Resource
-
+#
```

Comment:
```suggestion

```

Reply:
40d0d5510a7af0b0b1dbc942b38444a986ff058f

Reply:
?

Reply:
remove the extra #